C/other/Palindrome.c

```diff
@@ -0,0 +1,49 @@
+/*
+C Program to check whether a number is Palindrome or not
+An integer is a palindrome if the reverse of that number is equal to the original number.
+Example of Palindromes are 11,44,101,121,
+*/
+
+#include <stdio.h>
+#include <unistd.h>
+
+void main()
+{
+	int rev = 0, rem = 0, n = 0, num = 0;
```

Comment:
No need to initialize num and n as 0, you can just declare it as a normal variable.